src/components/Image/Image.tsx

```diff
@@ -6,8 +6,10 @@ import ImageElement from './ImageElement';
 type ImageProps = {
   alt: string;
   caption?: string;
+  captionVariant?: 'below' | 'right';
```

Comment:
`captionVariant` works but doesn't immediately communicate what the prop is for whereas `captionPosition` definitely would.

Reply:
Could argue that `captionVariant` allows for differences in style, that may be implemented based on the position?

Reply:
Ok